lib/beaker-hostgenerator/cli.rb

```diff
@@ -54,6 +54,16 @@ def initialize(argv = ARGV.dup)
    1 CentOS 6 64 bit node with roles = master, hypervisor = none, node name = static1, and my-key = my-value
    1 CentOS 6 32 bit node with roles = agent and the default hypervisor
 
+example with arbitrary host settings as a list:
```

Comment:
This should be indented over a couple spaces so it's aligned with the other examples